src/style.css

```
@@ -0,0 +1,45 @@
+* {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  background: #fff;
+  width: 100px;
+}
+
+.stars {
+  display: flex;
+  justify-content: flex-start;
+}
+
+.stars .stars__star {
+  display: flex;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars--1 :nth-child(1) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--2 :nth-child(-n+2) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--3 :nth-child(-n+3) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--4 :nth-child(-n+4) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--5 .stars__star {
+  background-image: url(images/star-active.svg);
+}
```

these styles can be merged
```suggestion
.stars--1 :nth-child(1),
.stars--2 :nth-child(-n+2),
.stars--3 :nth-child(-n+3),
.stars--4 :nth-child(-n+4),
.stars--5 .stars__star {
  background-image: url(images/star-active.svg);
}
```